code/controllers/subsystems/ticker.dm

```diff
@@ -413,13 +413,16 @@ var/datum/controller/subsystem/ticker/SSticker
 
 	var/can_start = src.mode.can_start()
 
-	if(can_start & GAME_FAILURE_NO_PLAYERS)
-		var/list/voted_not_ready = list()
-		for(var/mob/abstract/new_player/player in SSvote.round_voters)
-			if((player.client)&&(!player.ready))
-				voted_not_ready += player.ckey
+	var/list/voted_not_ready = list()
+	for(var/key in SSvote.round_voters)
+		var/mob/abstract/new_player/NP = new_player_list[key]
+		if(NP && !NP.ready)
```

Comment:
I think you still need to check for the client, right? They might disconnect but get ready.
```suggestion
		if(NP.client && !NP.ready)
```

Reply:
nah, i don't want people to DQ and dodge this check

Reply:
> nah, i don't want people to DQ and dodge this check

That is valid. But consider people who had to suddenly leave or something? You could like check how long since they DQ. If it has been long enough, not count them. Just some ideas

Reply:
i thought about this as well, but thankfully it's not an automated punishment system, so it doesn't have to be very smart. it's just a tool to give admins an idea of what's going on and who's doing this. they can do with that information whatever they want